### PR TITLE
#167727748 Allow users to receive notifications for posted comments

### PR DESCRIPTION
**Pull request by komsic**
## What does this PR do?

Enable users to receive notifications for posted comments

## Description of Task to be completed?

- modify notification type to include newComment
- edit create comment controller to create a notification after creating a new comment

## How should this be manually tested?
- CLONE this repo
- open the cloned directory
- run ``` npm install ```
- run ``` npm run start:dev ```
- access the docs using `{base-url}/docs`
- POST comment using `{base-url}/comments/:requestId`

## What are the relevant pivotal tracker stories?

[#167727748 ](https://www.pivotaltracker.com/story/show/167727748 )

### CODE REVIEW

File: src/services/notificationServices.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const sendViaApp = (id, title, body, actionLink) => client.trigger(`${id}`, 'not
  * @returns {null} - null
  */
 // eslint-disable-next-line object-curly-newline
-export const createNotification = async ({ sender, receiver, type, ref }) => {
+export const createNotification = async ({ sender, receiver, type, ref, inAppOnly }) => {
```
Comment (jsbuddy) [inline src/services/notificationServices.js]: Consider making `inAppOnly` a second argument instead.

File: src/services/notificationServices.js [hunk 2]
```diff
@@ -47,9 +47,11 @@ export const createNotification = async ({ sender, receiver, type, ref }) => {
     sender: sender.id, receiver: receiver.id, message: body, type, ref
   });
 
-  receiver.emailNotificationEnabled && sendViaEmail(
-    receiver.email, title, createEmailTemplate(title, body, actionLink, actionText)
-  );
+  if (!inAppOnly) {
+    receiver.emailNotificationEnabled && sendViaEmail(
```
Comment (jsbuddy) [inline src/services/notificationServices.js:51]: If block is not necessary here..  consider adding `!inAppOnly` to the short circuit.

**Comment by tejiri4** [comment]
Try your PR on heroku using the review app and add a comment if your endpoint is working on heroku

File: src/services/notificationServices.js (hunk 1 above)
Comment (komsic) [reply src/services/notificationServices.js]: I will make the changes

File: src/services/notificationServices.js (hunk 2 above)
Comment (komsic) [reply src/services/notificationServices.js:51]: I used the if statement to improve readability. Do you still think I should remove it?

File: src/services/notificationServices.js (hunk 1 above)
Comment (komsic) [reply src/services/notificationServices.js]: Done

File: src/services/notificationServices.js (hunk 2 above)
Comment (jsbuddy) [reply src/services/notificationServices.js:51]: Oh.. okay.. In that case, it's cool. Nice work!

**Comment by tejiri4** [comment]
Rebase your branch with develop

**Comment by komsic** [comment]
> Rebase your branch with develop

Done

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**